repositories.md

```diff
@@ -47,10 +47,7 @@ If the Time Server is implemented, the primary is CONDITIONALLY REQUIRED to use
 
 #### ECU Version Report
 
-The ECU version report from each Secondary will contain a token to be sent to the Time Server. To prevent a replay, this token SHOULD be unique for each update cycle. As we expect that these updates will be relatively infrequent (e.g., due to a limited number of write cycles), there will be a sufficient number of tokens to make this possible.
-
-
-The payload of the ECU version report sent to the Director may contain the token sent to the Time Server. This token is part of the version report sent from Secondaries to the Primary. If the token is removed, the signature will not match.
+The ECU version report from each Secondary will contain a token to be sent to the Time Server. To prevent a replay, each token SHOULD be unique per ECU for each update cycle. As we expect that these updates will be relatively infrequent (e.g., due to a limited number of write cycles), there will be a sufficient number of tokens to make this possible. The payload of the ECU version report sent from the Primary to the Director MAY contain the tokens sent to the Time Server. In this case, if any token is removed or changed, the signature will not match.
```

Comment:
As I understand https://github.com/uptane/deployment-considerations/pull/28#discussion_r339990698, one could rephrase this to indicate "One MUST send the tokens.  These tokens MAY be included in the ECU version report."  Is this accurate?   I think we have the second case covered, but not the first one.

Reply:
Isn't this already made clear in the preceding section? This is just describing how you can send them, but the requirement that they must be sent was already described.

Reply:
I think Justin's suggestion made this more clear. The idea is that you could send the tokens in a different way, but if you do it this way, you get the benefit of signature mismatches if something goes wrong. Is it clearer now that Justin's suggestion is merged?

Reply:
Yes. I think this is good to go. Should I merge?

Reply:
Up to you! I'm happy with it.

Reply:
OK. I'm merging.